CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ Check [Keep a Changelog](http://keepachangelog.com/) for recommendations on how
 
 ## [Unreleased]
 
+### Fixed
+- mirrord-agent: No longer resolves to `eth0` by default, now we first try to resolve the appropriate network interface, if this fails then we use `eth0` as a last resource. Fixes [#670](https://github.com/metalbear-co/mirrord/issues/670).
```

Comment:
Resort*

Reply:
Thank you! I was going crazy trying to remember this word!